File: atcoder/abc041_c.py
# https://atcoder.jp/contests/abc041/tasks/abc041_c

import numpy as np

N = int(input())
a = np.array(list(map(int, input().split())))
ans = np.argsort(a)[::-1] + 1

for i in ans:
    print(i)

File: atcoder/abc041_c.cpp
// https://atcoder.jp/contests/abc041/tasks/abc041_c
#include<iostream>
#include<algorithm>
//#include<string>
#include<vector>
//#include<map>
//#include<tuple>
//#include<queue>
//#include<regex>

// #include <bits/stdc++.h>
using namespace std;

int main() {
	int N;
	cin >> N;
	vector<pair<int, int>> a(N);
	for (int i = 0; i < N; i++)
	{
		int tmp;
		cin >> tmp;
		a[i] = make_pair(tmp, i +1);
	}

	sort(a.begin(), a.end());
	reverse(a.begin(), a.end());

	for (int i = 0; i < N; i++)
	{
		cout << a[i].second << endl;
	}

	return 0;
}
